bpf/lib/conntrack.h

```diff
@@ -39,6 +39,14 @@ static __always_inline bool conn_is_dns(__u16 dport)
 	return dport == bpf_htons(53);
```

Comment:
FYI, the commit `datapath/ct: terminate connections that were reset` triggers a checkpatch warning because it has two signed-off-by.

Reply:
yap, thanks! I've already fixed this into my local tree. I'll push it when I have all fixes ready.